content/actions/using-workflows/workflow-commands-for-github-actions.md

```diff
@@ -839,13 +833,13 @@ For multiline strings, you may use a delimiter with the following syntax.
 
 {% warning %}
 
-**Warning:** Make sure the delimiter you're using is randomly generated and unique for each run. For more information, see "[AUTOTITLE](/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections)".
+**Warning:** Make sure the delimiter you're using cannot occur on its own line in the value. If the value is completely arbitrary then don't use this format at all. Write to a separate file instead.
```

Comment:
I think it would be clearer to say:
```suggestion
**Warning:** Make sure the delimiter you're using won't occur on a line of its own within the value. If the value is completely arbitrary then you shouldn't use this format. Write the value to a file instead.
```